### PR TITLE
Hello Microverse project

### PR DESCRIPTION
**Pull request by beyk**
I fix linters and add a screenshot of the project to the Readme file

### CODE REVIEW

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Stylelint
+        run: |
+          npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+          [ -f .stylelintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css/.stylelintrc.json
+      - name: Stylelint Report
+        run: npx stylelint "**/*.{css,scss}"
```
Comment (FrederickMih) [inline .github/workflows/linters.yml:48]: - [ ] As mentioned by the previous reviewer, none of your linters are working properly, please let's fix it.
- [ ] This [Link](https://github.com/microverseinc/linters-config/tree/master/html-css) can help you to set up linters for this project. Kindly follow the instructions once more and you will be good to go!.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+## Built With
+- Html
+- Css
+
+## Authors
+👤 **Beyk**
+- GitHub: [@beyk](https://github.com/beyk)
+- Twitter: [@beyk_a](https://twitter.com/beyk_a)
+- LinkedIn: [Beyk](https://www.linkedin.com/in/asghar-beykmohammadi-1b16b291/)
```
Comment (FrederickMih) [inline README.md:15]: - [ ] Hi @beyk. 
I think it's better to add a screenshot of your app in the README file as mentioned by the previous reviewer. What do you think?. 

- [ ]  [OPTIONAL] Please modify [this template](https://github.com/microverseinc/readme-template) and add some instructions on how a non-technical person can set up the project locally in the `getting started section` :smiley: 


- [ ]  [OPTIONAL] You can equally consider adding the following sections to make your repo more professional: :+1: 

- **Contribution**
- **Show your support**
- **License**

**Review by FrederickMih — CHANGES_REQUESTED** [review]
Hi @beyk , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which you still need to address.

 ## Required Changes ♻️

- [ ] **We are supposed to maintain the same pull request for all Microverse projects**. Please don't close a PR when a project is not yet marked approve.

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by elmar8287** [comment]
Hi @beyk   , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉🎉🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**